packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-p2p-card/cashier-onboarding-p2p-card.tsx

```diff
@@ -15,10 +15,10 @@ const CashierOnboardingP2PCard: React.FC = observer(() => {
     const { openRealAccountSignup } = ui;
     const { setDepositTarget } = general_store;
     const history = useHistory();
+    const { data: is_has_p2p_supported_currencies } = useIsHasP2PSupportedCurrencies();
     const { data: is_p2p_enabled } = useIsP2PEnabled();
-    const has_usd_currency = useHasUSDCurrency();
     const has_fiat_currency = useHasFiatCurrency();
-    const should_show_p2p_card = is_p2p_enabled || has_usd_currency;
+    const should_show_p2p_card = is_p2p_enabled && is_has_p2p_supported_currencies;
```

Comment:
@farhan-nurzi-deriv Shouldn't it still be `or`? 🤔

Reply:
Actually that `or` was the cause of the error. So, if p2p is disabled but client has usd account, the card will still be shown which is not expected

Reply:
@farhan-nurzi-deriv But this was the requirement and expected 🤔 BE flag only checks for the current active account and if it's some currency that is not supported by P2P it will return p2p is disabled, So on the cashier onboarding page they wanted to show the p2p card if the user has any currencies which are supported by p2p so when they click on it we will switch the account 🤔 

Has the requirement changed? 🤔

Reply:
So, actually we need to hide the onboarding card for client from australia and new zealand. Even if they have the supported currency. cc: @farrah-deriv

Reply:
yup we want to hide the onboarding card now for australia and new zealand 👍

Reply:
@ameerul-deriv In that case we don't need to change `useIsP2PEnabled` behavior, Adding the new `useHasP2PSupportedCurrencies` will do the job 🤔 

Let's move the conversation to Slack 🙇🏻😬

Reply:
Yeah i think that should do 🤔 , i will try this